test/carryToken.js

```diff
@@ -0,0 +1,96 @@
+// The Carry token and the tokensale contracts
+// Copyright (C) 2018 Carry Protocol
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+const { assertEq, assertFail } = require("./utils");
+
+const CarryToken = artifacts.require("CarryToken");
+
+contract('CarryToken', async (accounts) => {
+    const instance = await CarryToken.deployed();
+    const [sender, receiver] = accounts;
+    const decimals = await instance.decimals();
+    const cre = new web3.BigNumber(10).pow(decimals);
+    const fromCre = cre.mul;
```

Comment:
This also can be destructured directly if the intermediate variable `cre` is not needed.

```js
const { mul: fromCre } = new web3.BigNumber(10).pow(decimals);
```